components/net/http_loader.rs

```diff
@@ -529,6 +530,25 @@ pub fn modify_request_headers(headers: &mut Headers,
     // https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch step 11
     if load_data.credentials_flag {
```

Comment:
@Manishearth made some notes about this on `#servo` [here-ish](http://logs.glob.uno/?c=mozilla%23servo&s=15+Feb+2016&e=15+Feb+2016#c365016), and in later WIP commits I made last night yet to be squashed and pushed to [i9547](https://github.com/danlrobertson/servo/tree/i9547), I did the basics of what was discussed on IRC.

I haven't dug into the "how" yet (I can look into this later), but in every build, run, and test I've ever done I've never seen `with_credentials != true`. I'm guessing it has something to do with [`SetWithCredentials`](https://github.com/servo/servo/blob/master/components/script/dom/xmlhttprequest.rs#L459-L474)?